pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -46,6 +46,12 @@ import (
 
 var bridgeFakeIP = "169.254.75.1%d/32"
 
+// the hardcoded MAC must be as low as possible, to prevent libvirt from
+// assigning a lower MAC to a tap device attached to the bridge - which
+// would trigger the bridge's MAC to update. This also applies for the
+// dummy connected to the bridge on masquerade binding.
```

Comment:
I am unable to recreate this on my `3.10.0-1160.6.1.el7.x86_64` kernel.
Once I set the bridge iface MAC, it is not changed if I attach a dummy.

Reply:
@AlonaKaplan do you remember the reference we had for this ?